tox.ini

```diff
@@ -4,9 +4,26 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27
+envlist = lint,py27
 
-[testenv]
+[testenv:lint]
+deps =
+    pylint
+    bandit
+
+commands =
+    # Run pylint, using securesystemslib's pylintrc configuration file
+    # https://github.com/secure-systems-lab/sample-documents/blob/master/pylintrc
+    pylint in_toto
+
+    # Run bandit, a security linter from OpenStack Security
+    # Note: We exclude tests that fail on importing or using the subprocess or
+    # subprocess32 modules, which are considered to have low severity security
+    # implications, but are required by in-toto. We do not skip subprocess
+    # calls with shell=True (B602).
+    bandit -r in_toto --skip B404,B603
```

Comment:
Can you link to B404 and B603, to make it easier for the reader to understand the implications of not testing them?

https://docs.openstack.org/bandit/latest/plugins/subprocess_without_shell_equals_true.html

https://docs.openstack.org/bandit/latest/plugins/subprocess_popen_with_shell_equals_true.html

https://docs.openstack.org/bandit/latest/blacklists/blacklist_imports.html#b404-import-subprocess